tools/release

```diff
@@ -118,7 +118,7 @@ fi
 
 ./tools/upload-release "$TARBALL"
 
-git push "$version"
+git push "$remote" "$version" --tags
```

Comment:
`git push "$remote" "$version"` suffices. I don’t think you generally want to push *all* tags.

Reply:
Good point. Generally, one wouldn't have tags that don't belong in the remote locally, but there's no reason this should push them if they happen to exist for whatever reason.